File: src/kyu_7/spoonerize_me/python/spoonerize_me_test.py
import codewars_test as test
from solution import spoonerize

@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(spoonerize("nit picking"), "pit nicking")
        test.assert_equals(spoonerize("wedding bells"), "bedding wells")
        test.assert_equals(spoonerize("jelly beans"), "belly jeans")
        test.assert_equals(spoonerize("pop corn"), "cop porn")

File: src/kyu_7/spoonerize_me/csharp/spoonerize_me_test.cs
using NUnit.Framework;

[TestFixture]
public class SpoonerizeTest
{
  [Test]
  public void NitPickingTest()
  {
    Assert.AreEqual("pit nicking", ReadySet.Spoonerize("nit picking"));
  }
  [Test]
  public void WeddingBellsTest()
  {
    Assert.AreEqual("bedding wells", ReadySet.Spoonerize("wedding bells"));
  }
  [Test]
  public void JellyBeansTest()
  {
    Assert.AreEqual("belly jeans", ReadySet.Spoonerize("jelly beans"));
  }
  [Test]
  public void PopCornTest()
  {
    Assert.AreEqual("cop porn", ReadySet.Spoonerize("pop corn"));
  }
}